src/vehicle_use.cpp

```diff
@@ -1130,6 +1130,31 @@ void vehicle::open_all_at( int p )
     }
 }
 
+bool vehicle::turn_on_internal_lights()
+{
+    bool success = false;
+
+    auto found = get_parts( VPFLAG_AISLE_LIGHT );
```

Comment:
Couldn't you search for both types with a lambda function, something like:
```
const auto turn_on_light_type = [&]( vpart_bitflags light_type ){
    bool success = false;
    auto light_parts = get_parts( light_type );
    for( vehicle_part *e  : found ) {
        e->enabled = true;
        success = true;
    }
    return success;
};

success |= turn_on_light_type( VPFLAG_AISLE_LIGHT ) || turn_on_light_type( VPFLAG_ATOMIC_LIGHT );
```

Reply:
Makes sense.  I'll do that.